src/components/City/style.css

```diff
@@ -0,0 +1,44 @@
+.cities {
+  margin: 10px;
+  display: flex;
+  flex-wrap: wrap;
+  align-items: center;
+  list-style: none;
+  justify-content: center;
+}
+
+.cities-card {
+  position: absolute;
+  bottom: 0;
+  height: 76%;
+  width: 100%;
+  background-color: #f0ecf1;
+  border-radius: 20px 20px 0 0;
+  padding-top: 4px;
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+}
+
+.city {
+  background-color: #00bbd6;
+  display: flex;
+  justify-content: center;
+  align-items: center;
+  margin: 5px;
+  border-radius: 20px;
+  width: 70%;
+  height: 100px;
+  font-size: 18px;
+}
+
+.city > * {
+  color: #e8e4ff;
+  text-decoration: none;
+}
+
+.city-details {
+  display: flex;
+  align-items: center;
+}
```

Comment:
- [ ] Kindly use `Lato` and `Gill Sans` fonts as `font-families` for the app according to the way they were used in the design template. 👍